proto/prehog/v1alpha/teleport.proto

```diff
@@ -1062,6 +1062,27 @@ message AuditQueryRunEvent {
   bool is_success = 3;
 }
 
+// DiscoveryFetchEvent is emitted when a DiscoveryService polls for new resources of a given type
+message DiscoveryFetchEvent {
+  // anonymized name of the DiscoveryGroup, 32 bytes (HMAC-SHA-256) encoded in base64
+  //
+  // PostHog property: tp.discovery_group
+  string discovery_group = 1;
```

Comment:
Based on  https://github.com/gravitational/cloud/pull/6833/files#diff-5f6d3d6dc4d22796cea6c4e045414a697376d9dccaab849e3fe2230373deb44cR274
in should be `tp.discovery.discovery_group` 😄